tools/FakeItEasy.Deploy/Program.cs

```diff
@@ -0,0 +1,104 @@
+﻿namespace FakeItEasy.Deploy
+{
+    using System;
+    using System.IO;
+    using System.Linq;
+    using System.Runtime.CompilerServices;
+    using System.Threading.Tasks;
+    using Octokit;
+    using static SimpleExec.Command;
+
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            var releaseName = Environment.GetEnvironmentVariable("APPVEYOR_REPO_TAG_NAME");
+            if (string.IsNullOrEmpty(releaseName))
+            {
+                Console.WriteLine("No Appveyor tag name supplied. Not deploying.");
+                return;
+            }
+
+            var (repoOwner, repoName) = GetRepositoryName();
+            var gitHubClient = GetAuthenticatedGitHubClient();
+
+            Console.WriteLine($"Deploying {releaseName}");
+            Console.WriteLine($"Looking for GitHub release {releaseName}");
+
+            var releases = await gitHubClient.Repository.Release.GetAll(repoOwner, repoName);
+            var release = releases.FirstOrDefault(r => r.Name == releaseName)
+                ?? throw new Exception($"Can't find release {releaseName}");
+
+            var artifactsFolder = Path.GetFullPath(Path.Combine(GetCurrentScriptDirectory(), "../../artifacts/output/"));
+            var artifactsPattern = "*.nupkg";
+
+            var artifacts = Directory.GetFiles(artifactsFolder, artifactsPattern);
+            if (!artifacts.Any())
+            {
+                throw new Exception("Can't find any artifacts to publish");
+            }
+
+            Console.WriteLine($"Uploading artifacts to GitHub release {releaseName}");
+            foreach (var file in artifacts)
+            {
+                await UploadArtifactToGitHubReleaseAsync(gitHubClient, release, file);
+            }
+
+            Console.WriteLine("Pushing nupkgs to nuget.org");

```

Comment:
It was. I could extract the nuget server URL to a constant (or maybe get it from an environment variable) and include the URL in this message, if you like.

Reply:
Or you could just say "NuGet". ;)

I'm not really bothered by any of thise.

Reply:
Well, I could say "NuGet", but it's convenient to be able to set the environment variable to a local folder for testing, and it would be scary to see "Pushing nupkgs to NuGet" if you do this :wink:

Reply:
But it might help someone reduce their :coffee: bill.